apps/server/src/modules/tool/controller/api-test/tool-school.controller.api.spec.ts

```diff
@@ -233,4 +233,40 @@ describe('ToolSchoolController (API)', () => {
 				});
 		});
 	});
+
+	describe('[GET] tools/school/:schoolExternalToolId', () => {
```

Comment:
You could write Params Validation Test for SchoolExternalToolIdParams

Reply:
The Params are checked elsewhere